src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/IProjectHotReloadSessionInternal.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.HotReload.Components.DeltaApplier;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload
+{
+    /// <summary>
+    /// Internal interface which Hot Reload sessions implement to provide access to the IDeltaApplier
+    /// </summary>
+    internal interface IProjectHotReloadSessionInternal
+    {
+        /// <summary>
+        /// Returns the delta applier for this session
+        /// </summary>
+        IDeltaApplier? DeltaApplier { get; }
```

Comment:
Could we just add this to `IProjectHotReloadSession` rather than creating a new interface?

Reply:
I'm fine doing that. I was just avoiding changing existing interfaces

Reply:
Actually, that is a public interface which is why didn't change it.